### ISSUE TITLE
When video has embedded audiodescription, don't enable button to upload one

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#22]
## Expected Behavior

When user mark that video **hasn't audio-description embedded**, the system enabled the upload button

This was working properly before please have a look at change history of the code.
How it work before
1. Initial state: Not embedded and audiodescription is required
Note that teacher needs to check if audiodescription is required or not
![image](https://user-images.githubusercontent.com/993369/76169545-0d841e00-6158-11ea-8b57-453e7de1f90b.png)
2. Embedded 
Note that everything else are disabled and accessibility resource is valid
![image](https://user-images.githubusercontent.com/993369/76169562-2a205600-6158-11ea-82df-26787bfebb0b.png)
3. Audio description is not required, accessibility resource is valid
![image](https://user-images.githubusercontent.com/993369/76169819-df540d80-615a-11ea-8292-25b6ffdc54c5.png)

See [working the code](https://gitlab.com/ecureuill/seli/-/blob/master/SELI-Platform/imports/components/accessibility/VideoAccessibilityForm.js#L139) 

NOTE: this was a simplified version for the prototype version of SELI. 
To fully attend WCAG requirement, see requirements #11 #12 #13  

## Current Behavior

When user mark that video **hasn't audio-description embedded**, the system enabled the upload button

## Possible Solution
Please refer to task section of requirement #11

**Comment by CAndresH** [comment]
Thanks Camila for the suggestions, I checked the previous code and found the required functionality, now it is enabled. I did not find the upload button with the style and icon that you show me in the screenshots, maybe you can share it, for now it is with the default button .

the link to the See working the code does not work.

According to your indications, I assume that the upload content icon is always available, but in the student part, the content will only be shown when audiodescription is checked or audiodescription is not checked and audiodescriptionrequired is checked,

I await your comments on the changes made on the platform

teacher part:
![image](https://user-images.githubusercontent.com/50594090/76647976-cfa64180-652b-11ea-978d-6fb7fb54d43b.png)

student part
![image](https://user-images.githubusercontent.com/50594090/76648123-1136ec80-652c-11ea-9410-4919a315f782.png)
